route/start.py

```diff
@@ -68,7 +68,6 @@ def __init__(self, name: str, target_state: TitleCursorPosition):
         self.target_state = target_state
 
     def execute(self, delta: float) -> bool:
-        title_sequence_manager.update()
         if title_sequence_manager._read_title_cursor_position() == self.target_state:
```

Comment:
"private" functions should not be called outside the class.

Also see comment on said function in `title_sequence_manager.py`